features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -137,6 +136,10 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
     const [ legacyPasswordPolicies, setLegacyPasswordPolicies ] = useState<ConnectorPropertyInterface[]>([]);
 
     const isReadOnly: boolean = !useRequiredScopes(featureConfig?.governanceConnectors?.scopes?.update);
+    const hasScopesForRuleBasedPasswordExpiry: boolean =
```

Comment:
Let's follow the convention.

```suggestion
    const hasRuleBasedPasswordExpiryReadPermissions: boolean =
```

Reply:
Changed